### PR TITLE
frontend: add commit hash inside footer

### LINKED ISSUE #505: Show commit/version in frontend
For debugging and bug reports it would help to display the version and/or commit hash in the frontend.

It's available on DO as a evn var: https://docs.digitalocean.com/products/app-platform/how-to/use-environment-variables/#app-wide-variables

# Acceptance criteria
- [ ] Commit version is visible on dapp and copyable

### PR DESCRIPTION
**Pull request by GabrielBuragev**
Closes #505

### CODE REVIEW

File: frontend/src/stores/transfer-history/store.ts [hunk 1]
```diff
@@ -17,34 +17,37 @@ export function getStoreName(appVersion: string): string {
   }
 }
 
-export const useTransferHistory = defineStore(getStoreName(getAppMajorVersion(APP_VERSION)), {
-  state: (): TransferHistoryState => ({
```
Comment (GabrielBuragev) [inline frontend/src/stores/transfer-history/store.ts:21]: might look like i've changed a lot here but its just due to reformatting. Only change is `APP_VERSION` -> `APP_RELEASE.VERSION`

**Review by compojoom — APPROVED** [review]
Code-wise LGTM.

Haven't checked the UI.

**Comment by GabrielBuragev** [comment]
Here is a screenshot with the UI @compojoom 

image

**Comment by istankovic** [comment]
Awesome, thanks for doing this! :champagne: 

BTW, what do you think about using `git describe --tags` to get the version? That typically shows more info,
like the tag information and how far we are from the tag, e.g.  `v1.0.1-85-gcaae8a3` (this means the version is 85 commits after the commit pointed to by `v1.0.1` tag, the `-g` part indicates `git` and `caae8a3` is the shortened SHA1 of the commit).

**Pull request merged**